### ISSUE TITLE
Issue with uuid-ossp

### ISSUE THREAD

**Issue by matthewford** [convox/postgres#4]
I have in my rails migration: `enable_extension 'uuid-ossp'`

Which throws the following error:

```
postgres | ERROR:  could not open extension control file "/usr/share/postgresql/extension/uuid-ossp.control": No such file or directory
postgres | STATEMENT:  CREATE EXTENSION IF NOT EXISTS "uuid-ossp"
```

Related: https://github.com/convox/postgres/pull/3

**Comment by jordaaash** [comment]
I found the issue. Alpine Linux has a package for postgresql-contrib, but the package is missing the uuid-ossp library: https://bugs.alpinelinux.org/issues/4832

@ddollar @csquared Is there a workaround for this without patching Alpine? It doesn't seem like the kind of fix that applied in #3 would work.

**Comment by ddollar** [comment]
Closed in favor of the Alpine upgrade issue
